src/backend/EdFi.DataManagementService.Backend.Mssql/EdFi.DataManagementService.Backend.Mssql.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+      <TargetFramework>net8.0</TargetFramework>
+      <Nullable>enable</Nullable>
+      <IsPackable>true</IsPackable>
+      <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+      <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="DbUp" Version="5.0.40" />
```

Comment:
Try switching from `dbup` to `dbup-core` and `dbup-postgresql`. I think `dbup` is just for compatibility with older .NET.

Reply:
That fixed it.